spec/features/review/user_edits_review_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+feature "remove a review for a restaurant", %q(
+  As a user I want to edit a review I added for a given restaurant
+) do
+
+  scenario "Authenticated user unsuccessfully edits a review they made",
+    focus: true do
+    restaurant = FactoryGirl.create(:restaurant)
+    user = FactoryGirl.create(:user)
+    sign_in_as(user)
+
+    review_owned_by_user = FactoryGirl.create(
+    :review, restaurant: restaurant, user: user
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.